lib/socket/socket.nit

```diff
@@ -147,6 +147,12 @@ class TCPStream
 		socket.write(msg.to_s)
 	end
 
+	redef fun write_byte(value)
+	do
+		if closed then return
```

Comment:
Maybe in this case you should set the `last_error` of the socket to an `IOError` of some kind if it was not set prior to this call.